constellations/gemini-user/src/error/error.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
+require("./error.sass");
+import React from "react";
+import { Link } from "react-router-dom";
+import srcBroken from "../../assets/compass_broken.png";
+
+export class ErrorPage extends React.Component {
```

Comment:
I would actually put this in the class folder since this is ONLY an error page related to classes. Would probably call this component ClassErrorPage